charts/sscsid-keeper/Chart.yaml

```diff
@@ -20,3 +20,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: changed
```

Comment:
you must remove ALL previous changes or they will be double

Reply:
Ahh okii